romana-install/roles/romana/install-agent/templates/config/romana.yaml

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
TLDR: "YAML can therefore be viewed as a natural superset of JSON"
http://yaml.org/spec/1.2/spec.html#id2759572